api/controllers/AuthController.js

```diff
@@ -68,12 +68,8 @@ module.exports = {
    * @param {Object} res
    */
   logout: function (req, res) {
-    req.logout();
-
-    // mark the user as logged out for auth purposes
-    req.session.authenticated = false;
-
-    res.redirect('/');
+    req.session.destroy();
```

Comment:
Alright, I'm assuming that req.session.destroy() also implicitly sets authenticated to false then?